packages/fxa-auth-server/config/index.ts

```diff
@@ -1979,6 +1979,26 @@ const convictConf = convict({
       format: String,
     },
   },
+  graphql: {
```

Comment:
Why `graphql` instead of `contentful`? Perhaps namespacing all of these Contentful config variables under `contentful` would make things a bit clearer?:

```ts
contentful: {
  apiKey: {
    // ...
  },
  apiUri: {
    // ...
  },
  environment: {
    // ...
  },
  spaceId: {
    // ...
  },
}
```

Also, this seems to be missing the `CONTENTFUL_GRAPHQL_API_URL` config? Looks like I missed it in the first config PR?

Reply:
Once these are all namespaced, the prefix of `contentful*` can be removed